### PR TITLE
R cmd build typo and doc clarifications

### PR DESCRIPTION
**Pull request by lcolladotor**
See https://github.com/r-hub/rhub/issues/52 for the discussion leading to this PR.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -28,7 +28,7 @@ Imports:
     utils,
     whoami,
     withr
-Suggests: 
+Suggests:
```
Comment (gaborcsardi) [inline DESCRIPTION]: PLease do not change this. Thanks!

File: R/check.R [hunk 2]
```diff
@@ -2,12 +2,14 @@
 #' Check an R package on r-hub
 #'
 #' @param path Path to a directory containing an R package, or path to
-#'   source R package tarball built with `R CMD check`.
+#'   source R package tarball built with `R CMD build` or \link{build}.
```
Comment (gaborcsardi) [inline R/check.R]: This must be `[devtools::build()]`. (`rhub` uses the new Markdown feature of the dev version of roxygen2.)

File: README.md [hunk 3]
```diff
@@ -89,7 +89,7 @@ platforms.
 ### Run a package check
 
 `check()` runs an `R CMD check` on the package in the specified directory,
-or specified source R package tarball. It defaults to the working
+or specified source R package tarball created by `R CMD build` or `build()`. It defaults to the working
```
Comment (gaborcsardi) [inline README.md]: PLease break long line (max ~75). Thanks!

File: DESCRIPTION (hunk 1 above)
Comment (lcolladotor) [inline DESCRIPTION]: That was probably automatic via `roxygen`

File: R/check.R (hunk 2 above)
Comment (lcolladotor) [inline R/check.R]: ok!

**Comment by lcolladotor** [comment]
Hi,

I just fixed my PR according to your comments. Let me know if you want me to squash the commit history as described in the second answer to http://stackoverflow.com/questions/7947322/preferred-github-workflow-for-updating-a-pull-request-after-code-review.

Best,
Leo

**Comment by gaborcsardi** [comment]
Thanks!

**Comment by lcolladotor** [comment]
No problem =)

**Pull request merged**